node/validation/manifest/schema.ts

```diff
@@ -18,7 +18,7 @@ const routesSchema = {
       type: 'string',
       format: 'regexPattern',
       errorMessage:
-        'pattern needs to be a regex that starts with ^ and ends with $ without any additional slashes before and afterwards',
+        'pattern needs to be a regex that starts with ^ followed by / and ends with $ without any additional slashes before and afterwards',
```

Comment:
I think these error messages need some love (perhaps we can ask @klavavej for some advice). But that can go in a follow-up PR.

Reply:
Thanks for the ping!

Is it possible to serve different error messages for issues in `netlify.toml` vs issues in `manifest.json`? I ask because ideally I'd like to provide a full example of the syntax for each to be super clear about the required format.

Error for issue in `netlify.toml`:

> pattern must be a regex that starts with ^/ and ends with $ for example pattern = "^/blog/[\d]{4}$"

Error for issue in `manifest.json`:

> pattern must be a regex that starts with ^/ and ends with $ for example "pattern": "^/blog/[\d]{4}$"

If that's not possible, we can use an example that focuses on just the value

> pattern must be a regex that starts with ^/ and ends with $ for example ^/blog/[\d]{4}$

Reply:
I start with one message for all, and we can look later if we can validate the path from the user config separately.